taar/settings.py

```diff
@@ -5,79 +5,81 @@
 
 from decouple import config
 
-PYTHON_LOG_LEVEL = config("PYTHON_LOG_LEVEL", "INFO")
 
-REDIS_HOST = config("REDIS_HOST", "localhost", cast=str)
-REDIS_PORT = config("REDIS_PORT", 6379, cast=int)
+class AppSettings:
+    PYTHON_LOG_LEVEL = config("PYTHON_LOG_LEVEL", "INFO")
+    STATSD_HOST = config("STATSD_HOST", default="localhost", cast=str)
+    STATSD_PORT = config("STATSD_PORT", default=8125, cast=int)
+    NO_REDIS = config("NO_REDIS", False, cast=bool)
```

Comment:
Can we change this to `ENABLE_REDIS` instead? I think it is easier to read.

Reply:
is `DISABLE_REDIS` ok? I assume that default behaviour is with redis enabled and we disable it for some specific cases, so `DISABLE_` feels more appropriate

Reply:
Works for me